inc/targetbase.class.php

```diff
@@ -1038,6 +1038,94 @@ protected function parseTags($content, PluginFormcreatorFormAnswer $formanswer,
       return $content;
    }
 
+   protected function showConditionSettings($rand) {
+      global $DB;
+
+      echo '<tr class="line0">';
+      
+      echo '<td width="15%">' . __('Condition') . '</td>';
+      echo '<td width="45%">';
+
+      $options[0] = __('No');
+      $options[1] = __('Yes');
+
+      $params['value'] = $this->fields['has_condition'];
+      $params['on_change'] = 'change_condition()';
+      $params['width'] = "65px";
+      $params['rand'] = $rand;
+
+      Dropdown::showFromArray("has_condition", $options, $params);
```

Comment:
GLPI provides a method to easily build a yes / no dropdown. This way you don't need to create $options array.